hubblestack/utils/signing.py

```diff
@@ -229,34 +307,44 @@ def __init__(self, public_crt, ca_crt):
                 continue
             already.add(digest)
             digest = digest.decode() + " " + stringify_ossl_cert(i)
-            log.debug('checking to see if %s is a valid leaf cert', digest)
+            log_level = log.debug
             try:
                 ossl.X509StoreContext(self.store, i).verify_certificate()
                 status = STATUS.VERIFIED
                 self.trusted.append(digest)
-                log.debug('  marking verified')
+                if check_verif_timestamp(digest, dampener_limit=seconds_day) == True:
```

Comment:
Redundant comparison to True, check_verif_timestamp() returns bool value already.